cmd/publish.go

```diff
@@ -0,0 +1,152 @@
+/*
+Copyright © 2024 Josh Holbrook <josh.holbrook@gmail.com>
+*/
+package cmd
+
+import (
+	"archive/tar"
+	"bytes"
+	"compress/gzip"
+	"fmt"
+	"io/fs"
+	"os"
+	"os/exec"
+	"path/filepath"
+
+	"github.com/rs/zerolog/log"
+	"github.com/spf13/cobra"
+
+	"github.com/jfhbrook/dosapp/config"
+	"github.com/jfhbrook/dosapp/pacman"
+)
+
+var publishCmd = &cobra.Command{
+	Use:   "publish",
+	Short: "Publish the current app",
+	Long:  `Publish the current app by creating and uploading a GitHub release.`,
+	Run: func(cmd *cobra.Command, args []string) {
+		cfg := config.NewConfig()
```

Comment:
It could be good to refactor this into its own library, even if it's one fatty function. But this is OK for now - it's not being reused, after all.